packages/ifpack2/adapters/thyra/RTOpPack_ROpGetSubVector_longdouble.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Teuchos_ConfigDefs.hpp"
+#include "RTOpPack_ROpGetSubVector_decl.hpp"
+
+#if defined(HAVE_RTOP_EXPLICIT_INSTANTIATION) && defined(HAVE_TEUCHOS_LONG_DOUBLE)
+
+#include "RTOpPack_ROpGetSubVector_def.hpp"
+#include "Teuchos_ExplicitInstantiationHelpers.hpp"
+
+namespace RTOpPack {
+
+TEUCHOS_CLASS_TEMPLATE_INSTANT_LONG_DOUBLE(ROpGetSubVector)
```

Comment:
@ikalash - is this a typo?  Should it be `RTOpGetSubVector` instead of `ROpGetSubVector`?

Reply:
Yes good catch!  This tells you that I was working on this in the evening on my laptop keyboard.  The "t" is messed up so that it either prints multiple t's or no t when I type it...  I will fix it.

Reply:
maybe not, the header file is consistent with the naming `RTOpPack_ROpGetSubVector_def.hpp`